pkg/ui/workspaces/cluster-ui/src/insights/workloadInsightDetails/transactionInsightDetailsOverviewTab.tsx

```diff
@@ -85,17 +85,23 @@ export const TransactionInsightDetailsOverviewTab: React.FC<Props> = ({
   );
 
   const blockingExecutions: ContentionEvent[] = contentionDetails?.map(x => {
+    const stmtInsight = statements.find(
+      y => y.statementExecutionID == x.waitingStmtID,
```

Comment:
nit: can we rename x and y to something a bit more descriptive, e.g. `stmt` or even just `s`